images/linux/scripts/installers/azpowershell.sh

```diff
@@ -6,92 +6,33 @@
 
 # Source the helpers for use with the script
 source $HELPER_SCRIPTS/document.sh
-source $HELPER_SCRIPTS/os.sh
 
-# Install Azure CLI (instructions taken from https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
-if isUbuntu20 ; then

```

Comment:
We'd like to install only the latest version for Ubuntu20. Why did you remove this section?

Reply:
Sorry, it was done by mistake. I reverted it back.

Reply:
You don't need this block then
![image](https://user-images.githubusercontent.com/48208649/83008738-52647600-a01e-11ea-9e69-5b3c0b5686c9.png)